client/src/components/BubblePage.js

```diff
@@ -1,17 +1,28 @@
 import React, { useState, useEffect } from "react";
-import axios from "axios";
+import { axiosWithAuth } from "../utils/axiosWithAuth";
 
 import Bubbles from "./Bubbles";
 import ColorList from "./ColorList";
 
-const BubblePage = () => {
+
+const BubblePage = (props) => {
+  // console.log('from BubblePage', props)
   const [colorList, setColorList] = useState([]);
   // fetch your colors data from the server when the component mounts
   // set that data to the colorList state property
+  useEffect(() => {
+    axiosWithAuth()
+    .get('http://localhost:5000/api/colors')
+    .then(res => {console.log('from useEffect Hook',res)
+      setColorList(res.data)
+    })
+    .catch(err => console.log(err.response))
+  },[])
+
 
   return (
     <>
-      <ColorList colors={colorList} updateColors={setColorList} />
+      <ColorList colors={colorList} updateColors={setColorList} props={props}/>
```

Comment:
what props are you passing?

Reply:
Yeah forgot to add props to the function itself, I needed access to props.match.params.id in the ColorList component to grab the id for each color... no wonder it was coming back undefined lol smh